runtime-modules/project-token/src/tests/fixtures.rs

```diff
@@ -111,6 +111,47 @@ impl IssueTokenFixture {
             ..self
         }
     }
+
+    pub fn with_split_rate(self, revenue_split_rate: Permill) -> Self {
+        Self {
+            params: IssuanceParams {
+                revenue_split_rate,
+                ..self.params
+            },
+            ..self
+        }
+    }
+
+    pub fn with_patronage_rate(self, patronage_rate: YearlyRate) -> Self {
+        Self {
+            params: IssuanceParams {
+                patronage_rate,
+                ..self.params
+            },
+            ..self
+        }
+    }
+
+    pub fn with_allocation(self, beneficiary_id: MemberId, amount: Balance) -> Self {
+        Self {
+            params: self.params.with_allocation(&beneficiary_id, amount, None),
+            ..self
+        }
+    }
+
+    pub fn with_empty_allocation(self) -> Self {
+        self.with_allocation(member!(1).0, 0u64.into())
+    }
```

Comment:
I tried adding some tests locally and I was confused why the allocation is `> 0` even when `with_empty_allocation` is used. Then I realized `with_allocation` only adds to existing allocation. How about making this method actually clear any existing allocation:
```suggestion
    pub fn with_empty_allocation(self) -> Self {
        Self {
            params: IssuanceParams {
                initial_allocation: BTreeMap::default(),
                ..self.params
            },
            ..self
        }
    }
```